src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Transform.cs

```diff
@@ -204,7 +204,7 @@ public Hashtable PropagatedNamespaces
                 CanonicalXmlNodeList namespaces = null;
                 if (reference != null)
                     namespaces = reference._namespaces;
-                else if (signedXml._context != null)
+                else if (signedXml?._context != null)
```

Comment:
@anthonylangsworth is this implementing this part correctly now? https://www.w3.org/TR/2001/REC-xml-c14n-20010315#NoNSPrefixRewriting - I'm not sure what PropagatedNamespaces should mean - I do not understand what does "namespace propagated into signature" is supposed to mean